<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceHistoryClass</class>
 <widget class="QWidget" name="SourceHistoryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>falconG - Gallery Source Selection</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="lvHistory"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>History size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbHistSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddDirectory">
         <property name="text">
          <string>&amp;Add</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="btnMoveUp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="falcong.qrc">
             <normaloff>:/icons/Resources/moveup-icon.png</normaloff>:/icons/Resources/moveup-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnMoveDown">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Resources/movedown-icon.png</normaloff>Resources/movedown-icon.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUndelete">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Undelete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSort">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>S&amp;ort</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelect">
         <property name="text">
          <string>&amp;Select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="falcong.qrc"/>
 </resources>
 <connections/>
</ui>
